cap/modules/schemas/utils.py

```diff
@@ -104,3 +112,27 @@ def is_later_version(version1, version2):
                 return False
             elif patch1 == patch2:
                 return False
+
+
+def validate_schema_config(config):
+    configs = ['notifications', 'reviewable', 'repositories']
+    all_errors = []
+
+    for name in configs:
+        with open(f'cap/modules/schemas/configs/{name}.json') as json_:
+            schema = json.load(json_)
+            data = config.get(name)
+            if not data:
+                continue
+
+            resolver = current_app.extensions['invenio-records']\
+                .ref_resolver_cls.from_schema(schema)
+            validator = DepositValidator(schema, resolver=resolver)
```

Comment:
we can just use default SchemaValidator from `jsonschemas` package, since they are static